source/module_gint/gint_k_pvpr.cpp

```diff
@@ -53,7 +53,7 @@ void Gint_k::destroy_pvpR(void)
     
     if(!pvpR_alloc_flag)
     {
-        ModuleBase::WARNING_QUIT("Gint_k::destroy_pvpR","<phi_0i | V | phi_Rj> matrix has not been allocated yet!");
```

Comment:
why are we removing this warning message?

Reply:
This is "warning and quit". If it's not allocated, terminating the program is overkill.